tools/editor/scenes_dock.cpp

```diff
@@ -959,6 +960,17 @@ void ScenesDock::_file_option(int p_option) {
 
 
 			String path = files->get_item_metadata(idx);
+			if (p_option == FILE_SHOW_IN_EXPLORER) {
+			  String dir = DirAccess::get_full_path("res://", DirAccess::ACCESS_FILESYSTEM);
+				const int res_begin = String("res://").length();
+				const int last_sep = path.find_last("/");
+				if (last_sep > res_begin) {
+					dir += "/";
+					dir += path.substr(res_begin, last_sep - res_begin);
+				}
+				OS::get_singleton()->shell_open(String("file://")+dir);
```

Comment:
I have tried to use `ACCESS_RESOURCES` it return a path begins with `res://` that can't be opened in native expolorer.